tests/test.sh

```diff
@@ -12,9 +12,9 @@ then
 elif [ "$SUITE" = "phpunit-full" ];
 then
     testcmd="bin/moodle-docker-compose exec -T webserver vendor/bin/phpunit --verbose"
-elif [ "$SUITE" = "app" ];
+elif [ "$SUITE" = "behat-app" ] || [ "$SUITE" = "behat-app-development" ];
 then
-    testcmd="bin/moodle-docker-compose exec -T webserver php admin/tool/behat/cli/run.php --tags=@app"
+    testcmd="bin/moodle-docker-compose exec -T webserver php admin/tool/behat/cli/run.php --tags='@app&&@mod_forum&&@app_from3.7'"
```

Comment:
Why is this leading to an execution with ZERO scenarios in the [travis build](https://travis-ci.org/github/moodlehq/moodle-docker/builds/703572428)? See the last 2 jobs, that are the ones introduced by this PR.

Reply:
I'm not sure why, maybe I did something wrong the last time. In any case, I've updated the code to use 3.9.0 and some tests are being run now.